Assets/HoloToolkit-Examples/InteractiveElements/Scripts/GestureInteractive.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using UnityEngine;
 using System.Collections;
 using HoloToolkit.Unity.InputModule;
+using UnityEngine;
+using Cursor = HoloToolkit.Unity.InputModule.Cursor;
 
-#if UNITY_EDITOR || UNITY_WSA
+#if UNITY_WSA || UNITY_STANDALONE
```

Comment:
UNITY_STANDALONE should probably be UNITY_STANDALONE_WIN, since I doubt the Windows Speech APIs will work on Mac or Linux :)